src/libs/conduit/conduit_node_iterator.cpp

```diff
@@ -135,6 +137,24 @@ NodeIterator::index() const
 }
 
 //---------------------------------------------------------------------------//
+std::string
+NodeIterator::id() const
```

Comment:
@xjrc, for some reason I thought that "name()" already did this logic, but I guess that wasn't the case?